### PR TITLE
Add back health check in the agent

### PR DESCRIPTION
**Pull request by huikang**
Signed-off-by: Hui Kang <kangh@us.ibm.com>

### CODE REVIEW

**Comment by huikang** [comment]
not sure if health check is necessary for agent. But when I deploy it in a k8s cluster, it maybe better to know if agent is running in a good state, so I add it back.

**Comment by ccaraman** [comment]
> not sure if health check is necessary for agent. But when I deploy it in a k8s cluster, it maybe better to know if agent is running in a good state, so I add it back.

I do think it is necessary. Health check is a good mechanism to know if some component is healthy. I don't think it matters if it is deployed standalone or as an agent.

File: examples/demo/docker-compose.yaml [hunk 1]
```diff
@@ -43,6 +43,7 @@ services:
       - "14268"       # Jaeger receiver
       - "55678"       # OpenCensus receiver
       - "55679:55679" # zpages extension
+      - "13133" # health_check
```
Comment (pjanotti) [inline examples/demo/docker-compose.yaml]: nit: use the same align as items above

Comment (huikang) [reply examples/demo/docker-compose.yaml]: updated. Thanks.

**Pull request merged**